app/controllers/all_galleries.py

```diff
@@ -3,13 +3,78 @@
 from werkzeug.security import check_password_hash
 from flask import session
 from app.models.Forms import *
+from app.models.Galleries import *
+from datetime import datetime
+
+
 
 @app.route('/', methods=["GET","POST"])
 def all_galleries():
-    user = Forms.get(fid=1)
-    name = user.first_name
-    print(name)
-    print(doesUserHaveRole("admin"))
-    return render_template('views/all_galleries.html')
+  
+    formGalleryIds=[] 
+    gallaryIds = []
+    
+    
+    map1={} #maps gallery ids and titles
```

Comment:
We should be able to do this using peewee and maybe some custom model methods.

Reply:
Noted